### DISCUSSION TITLE
polifill.io Malware?

### DISCUSSION THREAD

**Discussion by hhp94** [quarto-dev/quarto-cli#10283] [Q&A] [answered]
### Description

Hi,

I've been using RStudio to render my Quarto QMD document into HTML. Recently, I noticed that my HTML file is opening slower than usual. Upon checking my browser's console, I discovered it is due to "connecting to polyfill.io."

Given the recent vulnerabilities and malware issues associated with this domain (as reported in this [news article](https://thehackernews.com/2024/07/polyfillio-attack-impacts-over-380000.html)), I am concerned about potential exposure to this vulnerability. Can someone provide more information on this? I have no experience about cyber securities.

Thank you.

**Comment by mcanouil** [comment, marked as answer]
You need to upgrade your Quarto version.
As far as I know, >=1.4 no longer has this in the template coming from Pandoc.

- https://github.com/quarto-dev/quarto-cli/issues/10135

**Reply by mcanouil** [reply]
@hhp94 the usual way to deal with vulnerabilities is to upgrade the tool.

**Reply by hhp94** [reply]
Thank you so much. I had the Windows CLI installed. So I went to `https://quarto.org/docs/get-started/` to download the new version of the CLI and confirmed that it no longer connects to polifill.io. Thank you for your answer

**Reply by mcanouil** [reply]
In case you had the RStudio embedded version. Upgraded RStudio also use a more up to date version of Quarto (but not necessarily the latest contrary to a manual install).
